README.rst

```diff
@@ -8,17 +8,33 @@ directly to your Python code based on the types collected at runtime.
 Examples
 --------
 
+In the example, we have two files: ``some/module.py`` and ``myscript.py``.
+
+Say ``some/module.py`` originally contained::
+
+  def add(a, b):
+      return a + b
+
+And ``myscript.py`` contains::
+
+  from some.module import add
+
+  add(1, 2)
+
+Now we want to infer the type annotation of ``add`` in ``some/module.py`` by
+running ``myscript.py`` with ``MonkeyType``. This is supported in two ways:
+
 Run a script under call-trace logging of functions and methods in all imported
 modules::
 
   $ monkeytype run myscript.py
 
-Or enable call-trace logging for a block of code::
+Or enable call-trace logging for a block of code in ``myscript.py``::
 
   import monkeytype
 
   with monkeytype.trace():
-      ...
+      ...a block of code in myscript.py
```

Comment:
The `a block of code in myscript.py` comment is redundant. The previous sentence states that this code lives in `myscript.py`.

Reply:
Thanks for the review. Updated.